integration/spark/src/main/scala/org/apache/carbondata/spark/csv/CarbonCsvRelation.scala

```diff
@@ -148,6 +150,10 @@ case class CarbonCsvRelation protected[spark] (
           .withSkipHeaderRecord(false)
         CSVParser.parse(firstLine, csvFormat).getRecords.get(0).asScala.toArray
       }
+      if(null == firstRow) {
+        throw new DataLoadingException("Please check your input path and make sure " +
```

Comment:
May be csv file does not have header and user is passing the header from load command in that case is this a valid message??